docs/src/_sass/_normalize.scss

```diff
@@ -0,0 +1,341 @@
+/*! normalize.css v8.0.0 | MIT License | github.com/necolas/normalize.css */
```

Comment:
It'd be cool to pull this from a package manager instead (like we did when there was bower here). Not strictly needed, but we can spend a bit of time looking into it to see if there's some better, easy alternative.